projects/driver_controls/test/test_can_fsm.c

```diff
@@ -1,5 +1,7 @@
+// Test that the CAN events are generated and processed with the same event IDs and data bits
```

Comment:
I think this needs to be updated.

Reply:
Done

Reply:
It doesn't seem to have been updated?  `can_fsm` no longer exists - we should be verifying that each module produces reasonable data within their tests.